src/components/display/fewlines/DesktopNavigationBar/DesktopNavigationBar.tsx

```diff
@@ -26,7 +26,9 @@ export const DesktopNavigationBar: React.FC = () => {
       </ListItem>
       <ListItem>
         <LockIcon />
-        <ListLabel>Security</ListLabel>
+        <ListLabel onClick={() => router.push("/account/security")}>
```

Comment:
wait, is that a link?

Reply:
It is 😕. I'll create a story to refacto these into anchors, and tackle the issue you revealed above (i.e. _ I fear this render all links useless until react is loaded_).